Jenkinsfile

```diff
@@ -14,13 +14,13 @@
                 node('docker && highmem'){
                     checkout scm
                     def image = docker.build('jenkins/ath', 'src/main/resources/ath-container')
-                    image.inside('-v /var/run/docker.sock:/var/run/docker.sock') {
+                    image.inside('-v /var/run/docker.sock:/var/run/docker.sock --shm-size 2g') {
                         def exclusions = splits.get(index).join("\n");
                         writeFile file: 'excludes.txt', text: exclusions
                         realtimeJUnit(testResults: 'target/surefire-reports/TEST-*.xml', testDataPublishers: [[$class: 'AttachmentPublisher']]) {
                             sh '''
                                 eval $(./vnc.sh)
-                                ./run.sh firefox latest -Dmaven.test.failure.ignore=true -DforkCount=1 -B
+                                ./run.sh firefox latest -DargLine="-Djdk.net.URLClassPath.disableClassPathURLCheck=true" -Dmaven.test.failure.ignore=true -DforkCount=1 -B
```

Comment:
@jglick Points in #456 that we can just update surefire version to get the jdk fix see [here](https://github.com/jenkinsci/plugin-pom/pull/130) That seems better than having a temporary patch using a system property in this case

Reply:
Great, it is really better.